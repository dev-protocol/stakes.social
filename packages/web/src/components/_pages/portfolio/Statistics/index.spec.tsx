import React from 'react'
import { render } from '@testing-library/react'
import { Statistics } from '.'
import 'src/__mocks__/window/matchMedia.mock'

jest.mock('src/fixtures/dev-kit/hooks')
jest.mock('src/fixtures/uniswap/hooks')
jest.mock('@dev/graphql')

describe(`Statistics`, () => {
  test('Snapshot', () => {
    const component = render(<Statistics accountAddress="0xdummy" />)
    const tree = component.baseElement
    expect(tree).toMatchSnapshot()
  })
})
