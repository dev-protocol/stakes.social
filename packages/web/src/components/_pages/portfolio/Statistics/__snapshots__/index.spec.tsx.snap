// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Statistics Snapshot 1`] = `
<body>
  <div>
    <span
      class="ant-typography ant-typography-secondary"
    >
      (Not provide this feature yet on L2)
    </span>
  </div>
</body>
`;
